external/PATCH-01-STRATUM

```diff
@@ -2975,11 +2975,22 @@ index 9f4bb3a9..f07e4843 100644
 +       goto internal_error;
 +    }
  
++    auto keys_begin=keys.begin();
++    auto data_begin=data.begin();
++    std::advance(keys_begin,actual);
```

Comment:
We are iterating from begin to actual so correct naming convention would be keys_end and data_end